packages/fluentui/docs/src/examples/components/Dialog/Variations/DialogExampleZoomContent.shorthand.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Button, Dialog } from '@fluentui/react-northstar';
+import { useBooleanKnob } from '@fluentui/docs-components';
+import { CloseIcon } from '@fluentui/react-icons-northstar';
+
+const DialogExampleZoomContent: React.FC = () => {
+  const [open, setOpen] = useBooleanKnob({ name: 'open' });
+  const dialogVariables = {
+    rootWidth: '550px',
+  };
+  return (
+    <Dialog
+      open={open}
+      onOpen={() => setOpen(true)}
+      onCancel={() => setOpen(false)}
+      onConfirm={() => setOpen(false)}
+      variables={dialogVariables}
```

Comment:
Do you expect these variables to override the actual variables in `dialogVariables.ts` ? I don't think TMP uses variables in this way nor are they supposed to.

Reply:
That's a very good point, removed this from example.